internal/peering-request-operator/foreign-cluster.go

```diff
@@ -0,0 +1,129 @@
+package peering_request_operator
+
+import (
+	"errors"
+	v1 "github.com/liqoTech/liqo/api/discovery/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog"
+	"k8s.io/utils/pointer"
+	"strings"
+)
+
+func (r *PeeringRequestReconciler) UpdateForeignCluster(pr *v1.PeeringRequest) error {
+	tmp, err := r.crdClient.Resource("foreignclusters").List(metav1.ListOptions{
```

Comment:
as above, is it necessary to use `List` and then select the first one instead of directly use `Get`?

Reply:
As above